.github/workflows/ci.yaml

```diff
@@ -0,0 +1,44 @@
+---
+
+name: Continuous integration
+
+on:
+  push:
+
+jobs:
+  main:
+    name: Continuous integration
+    runs-on: ubuntu-18.04
+    timeout-minutes: 60
+    env:
+      DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+      DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+      TX_USR: ${{ secrets.TX_USR }}
+      TX_PWD: ${{ secrets.TX_PWD }}
+      TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+      TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+    steps:
+      - name: Install packages
+        run: sudo apt-get install -y gettext python3-virtualenv
+      - uses: actions/checkout@v1
+      - name: Build
+        run: make build
+      - name: Lint
+        run: make check
+      - name: Tests
+        run: make test
+      - name: Docker compose logs
+        if: ${{ always() }}
+        run: |
+          docker-compose logs
+          docker-compose down -v --remove-orphans
+      - name: Deploy PyPI
+        if: >
+          github.repository == 'camptocamp/c2cgeoform'
+          && success()
+          && (
+            startsWith(github.ref, 'refs/tags/')
+          )
+        run: |
+          sudo python3 -m pip install twine wheel
+          scripts/deploy-pypi
```

Comment:
Why not using c2cciutils to deploy?

Reply:
I've open a dedicated issue: https://github.com/camptocamp/c2cgeoform/issues/240